notebooks/signac_107_HOOMD-blue.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>maybe having an environment.yml in the repo with all the required packages (including jupyter) would be best?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Ah, my bad--there is an environment.yml. It's missing some of the notebook requirements though.

Reply:
I updated `environment.yml` in #25 and I was able to run all the notebooks from that environment. I hope this fixes whatever problem you saw!